### PR TITLE
Remove BENCHMARK_GUARDRAIL

### PR DESCRIPTION
**Pull request by nateberkopec**
[Anthropic's research](https://assets.anthropic.com/m/74342f2c96095771/original/Natural-emergent-misalignment-from-reward-hacking-paper.pdf) indicates that telling LLM's not to reward hack results in worse outcomes.

1. You put misaligned behavior into the context window, increasing salience. Putting the word "cheat" into context ~20 times per session will increase the probability the LLM is put into the "cheating" latent space.
2. GUARDRAIL lacks a definition of the real goal. "Don't overfit" and "don't cheat" doesn't mean a lot when you haven't defined what the system under test is and what it does.

We could go further than removing this guardrail, in the future. Ideas:

1. Survey user for details about the real production system under test, what conditions that system has to operate under.
2. Come up with a "hacking permitted" or even "hacking encouraged" prompt as described in the paper.

### CODE REVIEW

**Comment by davebcn87** [comment]
Interesting, the paper focuses on training not on executing. We noticed that it performs better with the prompt compared to before, but we haven't measured it in any scientific way. Got a back and forth using NotebookLM with the paper in it asking about that specific prompt. I also got this:

> if the developers applied "inoculation prompting" during the model's training phase (by telling the model that hacking was an acceptable, intended behavior for that specific training context), your prompt becomes highly beneficial during execution

So I think it might be model dependant.

**Comment by nateberkopec** [comment]
Another paper to consider might be the [assistant axis](https://www.anthropic.com/research/assistant-axis). Again, just my opinion, but inserting the words "overfit" and "cheat" so often into the context IMO pushes the agent off the actual axis of the task at hand and increases the salience of reward hacking.

Do you have public evals for autoresearch for testing changes like this?

**Comment by davebcn87** [comment]
Nice, thanks @nateberkopec I'll take a look at that paper as well. 

We don't run evals for it, but it would be a good way to test with/without that part of the prompt. I don't want to remove without backing data that proves is better, as it works quite good today.
